src/prefect/runner/storage.py

```diff
@@ -564,8 +564,72 @@ def __eq__(self, __value) -> bool:
         return False
 
 
-def create_storage_from_url(
-    url: str, pull_interval: Optional[int] = 60
+class LocalStorage:
+    """
+    Sets the working directory in the local filesystem.
+
+    Parameters:
+        Path: Local file path to set the working directory for the flow
+
+    Examples:
+        Sets the working directory for the local path to the flow:
+
+        ```python
+        from prefect.runner.storage import Localstorage
+
+        storage = LocalStorage(
+            path="/path/to/local/flow_directory",
+        )
+        ```
+    """
+
+    def __init__(
+        self,
+        path: str,
+        pull_interval: Optional[int] = 60,
+    ):
+        self._path = path
```

Comment:
When testing, I ran into errors caused by providing a relative path. I think that we'll want to use absolute paths internally. Here's one way to do that, but it will change how `self._path` is used elsewhere in this class.
```suggestion
        self._path = Path(path).resolve()
```

Reply:
Thanks 😃 this is done.